components/TreeHeatmap/index.js

```diff
@@ -119,16 +124,20 @@ export default class TreeHeatmap extends VisComponent {
 
     let keys = Object.keys(this.data[0]);
     let idColumn = this.idColumn;
-    if (!idColumn) {
+    if (idColumn === undefined) {
       idColumn = keys.includes('_id') ? '_id' : idColumn;
       idColumn = keys.includes('_') ? '_' : idColumn;
       idColumn = keys.includes('') ? '' : idColumn;
     }
+    if (idColumn === undefined) {
+      console.log('TreeHeatmap: No suitable idColumn found.');
```

Comment:
This is a condition where nothing can be rendered since there is no ID column specified, so yes I think this is fatal for initializing the visualization. I'll modify this to a throw and I think I can change the test to use `t.throws` to check for it.